app/controllers/users_controller.rb

```diff
@@ -7,6 +7,10 @@ def index
     @users = User.all
   end
 
+  def defualt
+    puts 'typo!'
```

Comment:
__[staging][RuboCop]__

Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.


<!-- Comment by [Sider](https://sider.review) -->